Weekly_Work/Week2/Metrics_Root_class.py

```diff
@@ -0,0 +1,98 @@
+import json
```

Comment:
This could be just "metric.py", or maybe "metric_df.py" (I think it would be a good idea to name it like the main class it has)

Reply:
Of course, will change that!